ethexe/cli/src/params/node.rs

```diff
@@ -98,7 +98,7 @@ impl NodeParams {
                 .with_context(|| "invalid `validator` key")?,
             validator_session: ConfigPublicKey::new(&self.validator_session)
                 .with_context(|| "invalid `validator-session` key")?,
-            max_commitment_depth: self.max_depth.unwrap_or(Self::DEFAULT_MAX_DEPTH).get(),
+            max_eth_sync_depth: self.max_depth.unwrap_or(Self::DEFAULT_MAX_DEPTH).get(),
```

Comment:
I'd rename cli param for parsing to also match the naming: so it will be max_sync_depth or --eth-max-sync-depth

Reply:
agree